2025-03-18 21:33:20,092 - WARNING - Retrying (Retry(total=6, connect=6, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7fbddbbceed0>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-18 21:33:35,083 - WARNING - Retrying (Retry(total=5, connect=5, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7fbe380e2840>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-18 21:33:53,189 - WARNING - Retrying (Retry(total=4, connect=4, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7fbe380e2540>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-18 21:34:20,006 - WARNING - Retrying (Retry(total=3, connect=3, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7fbe380e2cf0>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-18 21:35:02,586 - WARNING - Retrying (Retry(total=2, connect=2, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7fbe380e2f00>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-18 21:36:17,240 - WARNING - Retrying (Retry(total=1, connect=1, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7fbe380e2e70>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-18 21:36:39,208 - WARNING - Retrying (Retry(total=6, connect=6, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7fcc9b518500>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-18 21:36:53,920 - WARNING - Retrying (Retry(total=5, connect=5, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7fcc9a986690>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-18 21:37:12,491 - WARNING - Retrying (Retry(total=4, connect=4, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7fcc9a9de630>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-18 21:37:38,646 - WARNING - Retrying (Retry(total=3, connect=3, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7fcc9a9de750>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-18 21:38:21,502 - WARNING - Retrying (Retry(total=2, connect=2, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7fcc9a9dea80>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-18 21:39:35,989 - WARNING - Retrying (Retry(total=1, connect=1, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7fcc9a9deba0>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-18 21:41:45,998 - WARNING - Retrying (Retry(total=0, connect=0, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7fcc9a9de960>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-20 14:53:26,752 - WARNING - Retrying (Retry(total=6, connect=6, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f16f81169f0>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-20 14:53:41,717 - WARNING - Retrying (Retry(total=5, connect=5, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f17542ee300>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-20 14:54:00,551 - WARNING - Retrying (Retry(total=4, connect=4, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f17542eecf0>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-20 14:54:26,745 - WARNING - Retrying (Retry(total=3, connect=3, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f17542ef020>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-20 14:55:09,639 - WARNING - Retrying (Retry(total=2, connect=2, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f17542ef140>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-20 14:55:34,839 - WARNING - Retrying (Retry(total=6, connect=6, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f41ae7e09b0>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-20 14:55:49,717 - WARNING - Retrying (Retry(total=5, connect=5, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f41ab941370>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-20 14:56:08,503 - WARNING - Retrying (Retry(total=4, connect=4, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f41ab992c90>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-20 14:56:35,056 - WARNING - Retrying (Retry(total=3, connect=3, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f41ab992f30>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-20 14:57:17,860 - WARNING - Retrying (Retry(total=2, connect=2, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f41ab993050>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-20 14:58:32,695 - WARNING - Retrying (Retry(total=1, connect=1, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f41ab993290>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-20 15:00:13,158 - WARNING - Retrying (Retry(total=6, connect=6, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f5a741a0f50>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-20 15:00:27,961 - WARNING - Retrying (Retry(total=5, connect=5, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f5a74d047d0>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-20 15:00:46,754 - WARNING - Retrying (Retry(total=4, connect=4, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f5a741d2660>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-20 15:01:13,734 - WARNING - Retrying (Retry(total=3, connect=3, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f5a741d27e0>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-20 15:01:56,323 - WARNING - Retrying (Retry(total=2, connect=2, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f5a741d2ab0>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-20 15:03:23,043 - WARNING - Retrying (Retry(total=6, connect=6, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7fbb4345db50>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-20 15:03:37,133 - WARNING - Retrying (Retry(total=5, connect=5, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7fbb43969100>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-20 15:03:55,375 - WARNING - Retrying (Retry(total=4, connect=4, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7fbba02f6900>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-20 15:04:21,814 - WARNING - Retrying (Retry(total=3, connect=3, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7fbba02f6ba0>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-20 15:05:04,365 - WARNING - Retrying (Retry(total=2, connect=2, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7fbba02f6c00>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-20 15:06:19,404 - WARNING - Retrying (Retry(total=1, connect=1, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7fbba02f6f30>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-20 15:08:29,438 - WARNING - Retrying (Retry(total=0, connect=0, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7fbba02f7170>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500
2025-03-25 05:00:58,917 - INFO - Training script started.
2025-03-24 23:04:55,685 - INFO - Training script started.
2025-03-25 00:03:05,194 - INFO - Training script started.
2025-04-02 03:20:24,049 - INFO - Training script started.
2025-04-02 07:48:26,845 - INFO - Training script started.
2025-04-02 07:50:39,851 - INFO - Training script started.
2025-04-02 07:54:02,205 - INFO - Training script started.
2025-04-02 07:56:23,639 - INFO - Training script started.
2025-04-02 02:15:21,201 - INFO - Training script started.
2025-04-02 02:21:27,498 - INFO - Training script started.
2025-04-02 08:27:01,791 - INFO - Training script started.
2025-04-02 08:46:39,576 - INFO - Training script started.
2025-04-02 08:48:05,923 - INFO - Training script started.
2025-04-02 08:49:47,925 - INFO - Training script started.
2025-04-02 08:51:56,676 - INFO - Training script started.
2025-04-02 08:53:32,950 - INFO - Training script started.
2025-04-02 03:11:52,491 - INFO - Training script started.
2025-04-02 23:07:24,125 - INFO - Training script started.
2025-04-02 23:15:20,679 - INFO - Training script started.
2025-04-03 05:23:24,890 - INFO - Training script started.
2025-04-03 00:05:51,556 - INFO - Training script started.
2025-04-03 00:10:57,731 - INFO - Training script started.
2025-04-03 00:17:10,206 - INFO - Training script started.
2025-04-03 12:23:52,077 - INFO - Training script started.
2025-04-03 12:31:44,704 - INFO - Training script started.
2025-04-03 12:31:44,705 - ERROR - Failed to connect to MLflow server: module 'mlflow' has no attribute 'set_experience'
2025-04-03 12:31:44,705 - INFO - Ensure the MLflow server is running and the MLFLOW_TRACKING_URI is set correctly.
2025-04-03 12:34:16,945 - INFO - Training script started.
2025-04-03 12:36:48,068 - INFO - Training script started.
2025-04-03 12:57:11,210 - INFO - Training script started.
2025-04-03 19:50:33,894 - INFO - Training script started.
2025-04-03 19:55:12,648 - ERROR - Failed to connect to MLflow server: API request to http://mlflow:5000/api/2.0/mlflow/experiments/get-by-name failed with exception HTTPConnectionPool(host='mlflow', port=5000): Max retries exceeded with url: /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500 (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7fdc5d6a9e70>: Failed to resolve 'mlflow' ([Errno -2] Name or service not known)"))
2025-04-03 19:55:12,648 - INFO - Ensure the MLflow server is running and the MLFLOW_TRACKING_URI is set correctly.
2025-04-03 20:03:25,904 - INFO - Training script started.
2025-04-03 20:08:04,471 - ERROR - Failed to connect to MLflow server: API request to http://mlflow:5000/api/2.0/mlflow/experiments/get-by-name failed with exception HTTPConnectionPool(host='mlflow', port=5000): Max retries exceeded with url: /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500 (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7fcf504d5a50>: Failed to resolve 'mlflow' ([Errno -2] Name or service not known)"))
2025-04-03 20:08:04,472 - INFO - Ensure the MLflow server is running and the MLFLOW_TRACKING_URI is set correctly.
2025-04-03 20:35:28,238 - INFO - Training script started.
2025-04-03 20:40:23,624 - INFO - Training script started.
2025-04-03 20:45:02,011 - ERROR - Failed to connect to MLflow server: API request to http://mlflow:5000/api/2.0/mlflow/experiments/get-by-name failed with exception HTTPConnectionPool(host='mlflow', port=5000): Max retries exceeded with url: /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500 (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f586ad55fc0>: Failed to resolve 'mlflow' ([Errno -2] Name or service not known)"))
2025-04-03 20:45:02,011 - INFO - Ensure the MLflow server is running and the MLFLOW_TRACKING_URI is set correctly.
2025-04-03 20:48:23,051 - INFO - Training script started.
2025-04-03 20:52:54,618 - INFO - Training script started.
2025-04-03 20:54:30,030 - INFO - Training script started.
2025-04-03 20:59:07,616 - ERROR - Failed to connect to MLflow server: API request to http://mlflow:5000/api/2.0/mlflow/experiments/get-by-name failed with exception HTTPConnectionPool(host='mlflow', port=5000): Max retries exceeded with url: /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500 (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f3f6e21d4b0>: Failed to resolve 'mlflow' ([Errno -2] Name or service not known)"))
2025-04-03 20:59:07,617 - INFO - Ensure the MLflow server is running and the MLFLOW_TRACKING_URI is set correctly.
2025-04-03 22:43:31,433 - INFO - Training script started.
2025-04-03 22:48:09,456 - ERROR - Failed to connect to MLflow server: API request to http://mlflow:5000/api/2.0/mlflow/experiments/get-by-name failed with exception HTTPConnectionPool(host='mlflow', port=5000): Max retries exceeded with url: /api/2.0/mlflow/experiments/get-by-name?experiment_name=CMPT2500 (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f76fc7c1f00>: Failed to resolve 'mlflow' ([Errno -2] Name or service not known)"))
2025-04-03 22:48:09,456 - INFO - Ensure the MLflow server is running and the MLFLOW_TRACKING_URI is set correctly.
